src/app/components/practice/question-authoring-page/question-authoring-page.component.html

```diff
@@ -6,155 +6,296 @@
+                    <label>Hint 3</label>
+                    <input
+                      class=".mtdj__question-auth-input--long"
+                      clrInput
+                      formControlName="hint3"
+                      disabled
+                    />
+                  </clr-input-container>
+
+                  <clr-input-container id="mtdj__question-auth-input-option1">
+                    <label>Option 1</label>
+                    <input
+                      class=".mtdj__question-auth-input--long"
+                      clrInput
+                      formControlName="option1"
+                      disabled
+                    />
+                  </clr-input-container>
+                  <clr-input-container id="mtdj__question-auth-input-option2">
+                    <label>Option 2</label>
+                    <input
+                      class=".mtdj__question-auth-input--long"
+                      clrInput
+                      formControlName="option2"
+                      disabled
+                    />
+                  </clr-input-container>
+                  <clr-input-container id="mtdj__question-auth-input-option3">
+                    <label>Option 3</label>
+                    <input
+                      class=".mtdj__question-auth-input--long"
+                      clrInput
+                      formControlName="option3"
+                      disabled
+                    />
+                  </clr-input-container>
+                  <clr-input-container id="mtdj__question-auth-input-option4">
+                    <label>Option 4</label>
+                    <input
+                      class=".mtdj__question-auth-input--long"
+                      clrInput
+                      formControlName="option4"
+                      disabled
+                    />
+                  </clr-input-container>
+                </clr-tab-content>
+              </clr-tab>
+            </clr-tabs>
+
+            <button
+              type="submit"
+              class="btn btn-primary"
+              [disabled]="!newQuestionForm.valid"
+            >
               Submit Question
             </button>
-            <clr-alert
-              *ngIf="!validateOptions"
-              clrAlertType="danger"
-              [clrAlertClosable]="false"
-            >
-              Please make sure there is more than one option and they are in
-              order.
-            </clr-alert>
-            <clr-alert
-              *ngIf="questionExists"
-              clrAlertType="danger"
-              [clrAlertClosable]="false"
-            >
-              A Question of that title already exists.
-            </clr-alert>
-            <clr-alert
-              *ngIf="submitted"
-              clrAlertType="success"
-              [clrAlertClosable]="false"
-            >
-              Question submission successful.
-            </clr-alert>
           </form>
         </div>
         <div
```

Comment:
On a higher level design concept, I think adding a way to upload images on this form is definitely something for the future.